index.js

```diff
@@ -4,9 +4,11 @@ var registryUrl = require('registry-url');
 var rc = require('rc');
 
 module.exports = function (name, version) {
+	var isScoped = name.indexOf('/') > -1;
 	var scope = name.split('/')[0];
 	var url = registryUrl(scope) +
-		encodeURIComponent(name).replace(/^%40/, '@');
+		encodeURIComponent(name).replace(/^%40/, '@') +
+		(!isScoped && version ? '/' + version : '');
```

Comment:
@sindresorhus I'm using semver module in the PR #17, see: https://github.com/sindresorhus/package-json/pull/17/files